### ISSUE TITLE
Allow editing the description / private notes of a transaction

### ISSUE THREAD

**Issue by veqtrus** [bitcoin-solutions/multibit-hd#694]

**Comment by jim618** [comment]
I think this would be useful - we've made them read-only at the moment mainly as it was simpler to do.

**Comment by gary-rowe** [comment]
Agreed. I don't think we need to create an audit trail of corrections/modifications so just a simple overwrite during persistence should be sufficient.

Bumping to 0.1.4.

**Comment by mikehearn** [comment]
Yes please! I just tried to do this and was confused when it wasn't possible.

You might want to make the tx details view a tab pane instead of requiring the user to click next four times. That's not really a wizard style UI.

**Comment by gary-rowe** [comment]
@mikehearn Yeah, we did think about the Tx details being a bit more accessible, but most users don't generally dig into their transactions unless there's something seriously wrong so it's mostly a rare event. Not sure that putting it higher in the mix is worth it for such an infrequent operation (for most users).

**Comment by gary-rowe** [comment]
Assigning this to 0.2.1.

**Comment by gary-rowe** [comment]
Had quite a bit of feedback about this, so I'm bumping it as a priority.

**Comment by ebonit** [comment]
great thanks!

**Comment by t4777sd** [comment]
Has this been abandoned? The 0.3 version has come and gone

**Comment by gary-rowe** [comment]
Control of the multibit-hd repo has changed hands and so several issues were moved into the backlog for re-assessment prior to the handover.
